test-network-function/lifecycle/suite.go

```diff
@@ -329,26 +342,11 @@ func testPodsRecreation(env *config.TestEnvironment) {
 				log.Debug("node ", n.Name, " has no deployment, skip draining")
 				continue
 			}
-			// We need to delete all Oc sessions because the drain operation is often deleting oauth-openshift pod
-			// This result in lost connectivity oc sessions
-			env.ResetOc()
```

Comment:
@edcdavid You added this. Not sure why it's inside the loop?

Reply:
This is to prevent the broken sessions issues. we close all the sessions before they are potentially  disconnected

Reply:
Missed your initial point. Yes, this call should be made only once before starting to drain all the nodes.

Reply:
Should I move it outside the loop, then?

Reply:
yes

Reply:
Done already.